src/outputs/parthenon_hdf5.hpp

```diff
@@ -307,7 +211,7 @@ void HDF5ReadAttribute(hid_t location, const std::string &name, T &view) {
   auto *pdata = view.data();
   auto view_h = Kokkos::create_mirror_view(view);
   if constexpr (!std::is_same<typename T::memory_space, Kokkos::HostSpace>::value) {
-    Kokkos::deep_copy(view_h, view);
+    //Kokkos::deep_copy(view_h, view); // JMM: Pretty sure this deep copy is unnecessary
```

Comment:
Are you sure? I thought `Kokkos::create_mirror_view` populated the new view with zeros and so the `deep_copy` is necessary

Reply:
Alternatively, there's a `Kokkos::create_mirror_view_and_copy` which should handle the case automatically.

Reply:
This call is made to check type before reading the param... so I believe a mirror view is required, but copying is not needed. That said, it doesn't hurt (other than maybe hurting performance/compile times slightly). I think if restart tests pass, we can assume we don't need this deep copy. But if you're nervous I'm happy to put it back in.